google/cloud/aiplatform/datasets/image_dataset.py

```diff
@@ -0,0 +1,178 @@
+                Storage describing the import format. Validation will be
+                done against the schema. The schema is defined as an
+                `OpenAPI 3.0.2 Schema
+                Object <https://tinyurl.com/y538mdwt>`__.
+            data_item_labels: Optional[Dict] = {}
+                Labels that will be applied to newly imported DataItems. If
+                an identical DataItem as one being imported already exists
+                in the Dataset, then these labels will be appended to these
+                of the already existing one, and if labels with identical
+                key is imported before, the old label value will be
+                overwritten. If two DataItems are identical in the same
+                import data operation, the labels will be combined and if
+                key collision happens in this case, one of the values will
+                be picked randomly. Two DataItems are considered identical
+                if their content bytes are identical (e.g. image bytes or
+                pdf bytes). These labels will be overridden by Annotation
+                labels specified inside index file refenced by
+                [import_schema_uri][google.cloud.aiplatform.v1beta1.ImportDataConfig.import_schema_uri],
+                e.g. jsonl file.
+            project: Optional[str] = None,
+                Project to upload this model to. Overrides project set in
+                aiplatform.init.
+            location: Optional[str] = None,
+                Location to upload this model to. Overrides location set in
+                aiplatform.init.
+            credentials: Optional[auth_credentials.Credentials] = None,
+                Custom credentials to use to upload this model. Overrides
+                credentials set in aiplatform.init.
+            request_metadata: Optional[Sequence[Tuple[str, str]]] = ()
+                Strings which should be sent along with the request as metadata.
+            sync: (bool) = True
+                Whether to execute this method synchronously. If False, this method
+                will be executed in concurrent Future and any downstream object will
+                be immediately returned and synced when the Future has completed.
+
+        Returns:
+            image_dataset: ImageDataset
+                Instantiated representation of the managed image dataset resource.
+
+        """
+
+        utils.validate_display_name(display_name)
+        cls.metadata_schema_uri = schema.dataset.metadata.image
+
+        datasource = NonTabularDatasource()
+        import_data_config = None
+        if import_schema_uri:
+            cls.import_schema_uri = import_schema_uri
+            datasource = NonTabularDatasourceImportable(
+                gcs_source, import_schema_uri, data_item_labels
+            )
+            import_data_config = datasource.import_data_config
+
+        return cls._create_and_import(
```

Comment:
I thought we agreed to create a function that encapsulates Tabular/Non-tabular parameters as discussed in https://docs.google.com/document/d/16Nu_jFnGO79mcn83IgtpvTCYYEwxyPLOgdvxdxfILbI/edit?resourcekey=0-89933HPbKWlN6sz31SvBuA#bookmark=id.k68tqmbzwags

That way you wouldn't `_create_and_import` directly.

Reply:
We can discuss offline.